terraform/aws/inventory.tmpl

```diff
@@ -6,6 +6,7 @@ all:
     cluster_ip: ${cluster_ip}
     aws_stonith_tag: ${stonith_tag}
     aws_region: ${region}
+    ebs_id_to_device_name: ${jsonencode(ebs_map)}
   children:
```

Comment:
here an example of how this line looks like when resolved in an Ansible inventory.yaml file (from PR VRs) https://openqaworker15.qa.suse.cz/tests/324314/logfile?filename=serial_terminal.txt#line-1726